examples/TEM_data/entities/Chemistry.json

```diff
@@ -0,0 +1,53 @@
+{
+  "uri": "http://onto-ns.com/meta/0.3/Chemistry",
+  "description": "Simple entity with alloy and particle compositions.",
+  "dimensions": {
+    "nelements": "Number of different chemical elements.",
+    "nphases": "Number of phases."
+  },
+  "properties": {
+    "alloy": {
+      "type": "string",
+      "description": "A human description of the alloying system and temper."
+    },
+    "elements": {
+      "type": "string",
+      "shape": ["nelements"],
+      "description": "Chemical symbol of each chemical element.  By convension the dependent element (e.g. Al) is listed first."
```

Comment:
What does 'dependent element' mean?

Reply:
When specifying the composition of an alloy, it is sufficient to specify the amount of all elements but one. E.g. for an Al-Mg-Si alloy with 0.3% Mg and 0.3% Si, Al is the dependent element, who's amount can be calculated from the others (since the amounts should sum to 100%).

Reply:
I understand. I am not used to describing alloys like this, but I am also not an alloy expert. I do not think this is terribly important for the demo, but I was curious and a google search on 'dependent element in alloy' only leads me to a code you have made at ntnu. 

As mentioned, this is not important. Your choice as you are the expert.